.travis.yml

```diff
@@ -10,5 +10,6 @@ env:
     - ROBOT_ENVIRONMENT=test
 
 rvm:
-  - 2.3.7 # deployed
+  - 2.3.7
```

Comment:
Why keep these around if we aren't deploying on these versions?

Reply:
yeah, i'll ditch 2.3.7